samples/visual_indicator/nodejs/README.md

```diff
@@ -0,0 +1,128 @@
+---
+page_type: sample
+description: This sample app shows how to send visual indicator in meetings using targeted notifications API.
```

Comment:
Why you have pushed 2 samples ????

Reply:
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/59052547/67c571c4-94f7-4b33-a3f8-a3b82a4664bb)